MAINTAINERS.md

```diff
@@ -1,18 +1,39 @@
 # Laminas Project Maintainers
 
+Maintainers are trusted contributors who have GitHub's `maintainer` rights to one or more repositories under our GitHub organizations.
+
+Beyond driving fixes/improvements, we expect maintainers to:
+
+- Ensure project continuity, quality, and consistency
+- Review and merge relevant code modifications
+- Triage issues
+- Release new versions
+- Participate in technical discussions and decision-making
+
 We aim to have 2+ maintainers per repository.
-Maintainers have GitHub maintainer rights to the repository(ies) they maintain, which means the ability to push code to the repository, manage issues and pull requests, and manipulate selected repository settings.
 
-To become a maintainer, you must first have another maintainer or a TSC member nominate you to the TSC.
-Nominations are done by submitting an issue to this repository with a title prefix of `[NOMINATION][MAINTAINER]` (or select the "Maintainer Nomination" issue type when creating an issue on this repository).
-Maintainers are approved by a majority vote of the TSC, which will be held no later than 1 week following the nomination.
+## Current maintainers
 
 Current maintainers are found on the Teams page of each organization, on a team matching the repository name:
 
 - [Laminas organization teams](https://github.com/orgs/laminas/teams)
 - [Laminas API Tools organization teams](https://github.com/orgs/laminas-api-tools/teams)
 - [Mezzio organization teams](https://github.com/orgs/mezzio/teams)
 
+## Becoming a Maintainer
+
+### Requirements
+
+- Consistent high-quality contributions (code changes, reviews, discussions, etc) over time
```

Comment:
Can we make "over time" a little more explicit?

Reply:
Who evaluates that contributions are of "high-quality"? Should it be the nominator's duty to validate that before submitting the nomination?

Reply:
It would be nice to know in advance that they will not quit after one month but you cannot expect fidelity.  No one can expect fidelity of any employee in any company, so even less from volunteers in an OSS project.  

However, a track record of contributions, consistent over time, would give a good warm and fuzzy feeling that the person would stick around.
  
I propose the following language:
- Consistent high-quality contributions (code changes, reviews, discussions, etc) on a regular basis